test/function/samples/can-import-self/main.js

```diff
@@ -0,0 +1,3 @@
+import me from './main';
```

Comment:
I think we should change this example to something a little closer to an actual use-case. How about something like this:

```javascript
// main.js
import {run} from './lib.js';
run();

// lib.js
import * as lib from './lib';
export const run = () => lib.log();
export const log = () => console.log('log');
```

? This would document a little better as to why someone would even want to do that. And we can also remove the checked warning from the config which I think is probably rather distracting to the unsuspecting reader.

And maybe we could also make the "empty" test a little more interesting by basically doing the same but without a side-effect in the secondary function. Then we can show that tree-shaking works properly as well in the case of self-imports.

What do you think?